editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -574,15 +683,69 @@ export const editorStore: StateCreator<
     }
   },
 
-  pasteNode(toParent, toBefore) {
+  pasteClonedNode(toParent, toBefore) {
     try {
-      const id = localStorage.getItem("clipboard");
+      const id = get().getClonedNodeId();
       if (id) {
         const [, ops] = clone(id, { toParent, toBefore })(get().flow);
         send(ops);
       }
-    } catch (err: any) {
-      alert(err.message);
+    } catch (err) {
+      alert((err as Error).message);
+    }
+  },
```

Comment:
Question: If trying to "clone" _across_ flows, could we default to "copy" behaviour rather than disable "paste" option all together as now? 

From the current context menu, I'm not sure it's so clear which action is available _in this flow and across flows_ (copy) versus _in this flow only_ (clone) - and maybe rather than figuring out a way to explain that we could just have a smarter fallback behaviour?